src/Chapter2.hs

```diff
@@ -780,14 +788,13 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
--- TODO: type ;)
-listElementsLessThan x l = filter (< x) l
 
--- Can you eta-reduce this one???
-pairMul xs ys = zipWith (*) xs ys
+listElementsLessThan x = filter (< x)
 
+-- Can you eta-reduce this one???
+pairMul = zipWith (*)
```

Comment:
Eta reduction award 🏆 

Also, we advise adding type signatures to all functions 🙂